### PR TITLE
Fix url error in menu-item

### PR DESCRIPTION
**Pull request by HanielF**
Fix link error in menu items when multi-languages is disabled.

### CODE REVIEW

File: layout/_partials/header/menu-item.njk [hunk 1]
```diff
@@ -3,7 +3,7 @@
   {%- set itemURL = value.split('||')[0] | trim %}
   {%- if not itemURL.startsWith('http') %}
     {# set default links for multilanguage support #}
-    {%- if config.language[0] == page.lang %}
+    {%- if config.language.length == 2 or config.language[0] == page.lang %}
```
Comment (Gabirel) [inline layout/_partials/header/menu-item.njk:6]: `config.language.length > 1` is more practical for multilingual users. What do you think?

Comment (HanielF) [reply layout/_partials/header/menu-item.njk:6]: `config.language` will be`[ en, default]` if the lang is set to en.

Comment (HanielF) [reply layout/_partials/header/menu-item.njk:6]: You can print the value to verify.

Comment (Gabirel) [reply layout/_partials/header/menu-item.njk:6]: I see. Thanks for your explanation.

**Review by Gabirel — APPROVED** [review]

**Pull request merged**